src/main/java/org/example/servlets/FeedbackAddServlet.java

```diff
@@ -0,0 +1,65 @@
+package org.example.servlets;
+
+import org.example.HibernateUtil;
+import org.example.entity.Advertisement;
+import org.example.entity.Feedback;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+import static java.lang.Integer.parseInt;
+
+public class FeedbackAddServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        req.setAttribute("id", req.getParameter("id"));
+        getServletContext().getRequestDispatcher("/WEB-INF/jsp/feedback.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        Feedback feedback = new Feedback();
+        LocalDateTime date = LocalDateTime.now(ZoneId.of("Europe/Minsk"));
+        try {
+            SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+            Session session = sessionFactory.openSession();
+            Transaction t = session.beginTransaction();
+            Feedback paramId = session.load(Feedback.class, Integer.parseInt(req.getParameter("id")));
+            Advertisement ad = session.load(Advertisement.class, parseInt(req.getParameter("id")));
```

Comment:
You load 2 different beans by the same parameter. It seems, something don't work)